app/views/fact/isbn.html.erb

```diff
@@ -2,13 +2,13 @@
 
 <div id="isbn-fact" class="panel panel-info fact">
   <div class="panel-heading">
-    <%= @json[:title] %>
+    <%= link_to @json[:title], @json[:openurl] %>
```

Comment:
Non-blocking idea:

In addition to this title link, we might want to also add (as a replacement for the unimplemented "yes/not relevant links at the bottom of the panel), something like the DOI panel adds for consistency across panels (the specific wording feels weird for ISBN though so maybe `Check MIT Access||Availability`? Not sure.

```ruby
<li><%= link_to('Check MIT Subscription Access', @json[:link_resolver_url]) %>
```

Reply:
Yeah I'm not sure we have a clear idea here. I had put those in at some point to demonstrate the idea about feedback so they may come back. I'm totally fine with either direction on this fwiw.